examples/js-chart-and-grid-messagerouter/js-chart/webpack.config.cjs

```diff
@@ -0,0 +1,40 @@
+// Generated using webpack-cli https://github.com/webpack/webpack-cli
```

Comment:
is the header needed here?

Reply:
As we discussed this is a config file, so we don't need to add a license header here.